pkg/controller/scheduledjob/controller.go

```diff
@@ -203,8 +203,7 @@ func SyncOne(sj batch.ScheduledJob, js []batch.Job, now time.Time, jc jobControl
 		return
 	}
 	if sj.Spec.ConcurrencyPolicy == batch.ReplaceConcurrent {
-		for i := range sj.Status.Active {
-			j := sj.Status.Active[i]
```

Comment:
Aside from my dumbness :stuck_out_tongue_winking_eye: , if you look through the code, I'm removing objects from `sj.Status.Active` as one of the last operations in that loop. For-range does [in-place copy](http://stackoverflow.com/questions/15945030/change-values-while-iterating-in-golang/15952415#15952415) of all objects so it's safe to remove them from the list.